RecoMET/METProducers/src/METSignificanceProducer.cc

```diff
@@ -0,0 +1,108 @@
+// -*- C++ -*-
+//
+// Package:    METProducers
+// Class:      METSignificanceProducer
+//
+//
+
+//____________________________________________________________________________||
+#include "RecoMET/METProducers/interface/METSignificanceProducer.h"
+
+
+//____________________________________________________________________________||
+namespace cms
+{
+
+//____________________________________________________________________________||
+  METSignificanceProducer::METSignificanceProducer(const edm::ParameterSet& iConfig)
+  {
+    
+    std::vector<edm::InputTag> srcLeptonsTags = iConfig.getParameter< std::vector<edm::InputTag> >("srcLeptons");
+    for(std::vector<edm::InputTag>::const_iterator it=srcLeptonsTags.begin();it!=srcLeptonsTags.end();it++) {
+      lepTokens_.push_back( consumes<edm::View<reco::Candidate> >( *it ) );
+    }
+    
+    pfjetsToken_    = consumes<edm::View<reco::Jet> >(iConfig.getParameter<edm::InputTag>("srcPfJets"));
+
+    metToken_ = consumes<edm::View<reco::MET> >(iConfig.getParameter<edm::InputTag>("srcMet"));
+    pfCandidatesToken_ = consumes<edm::View<reco::Candidate> >(iConfig.getParameter<edm::InputTag>("srcPFCandidates"));
+
+    metSigAlgo_ = new metsig::METSignificance(iConfig);
+
+   produces<double>("METSignificance");
+   produces<double>("CovarianceMatrix00");
+   produces<double>("CovarianceMatrix01");
+   produces<double>("CovarianceMatrix10");
```

Comment:
Yes that makes more sense to store the matrix.
However, it asks for the Reflex dictionnary of 
edm::Wrapper<ROOT::Math::SMatrix<double,2u,2u,ROOT::Math::MatRepStd<double,2u,2u> > >
which is not generated when added in the corresponding classes_def.xml

I did not find any hint of an equivalent dictionnary in lxr to see what could be the problem. Would you have a suggestion?